src/utils/router_state.rs

```diff
@@ -310,8 +327,7 @@ pub fn init_router() -> &'static str {
             |from, to| haversine::distance(&from.as_node().location, &to.as_node().location),
             |from, to| haversine::distance(&from.as_node().location, &to.as_node().location),
         ))
-        .expect("Failed to initialize router");
-    "Arrow Cargo router initialized."
```

Comment:
Why remove these?

Reply:
I changed the return type of the function init_router to be `Result<(), String>` instead of `&'static str`.
So I assumed when things go OK, we don't need to return message, maybe we can just log message instead. But if there is an error, that's when we need to get message.

Reply:
Also, when possible (when function returns result or option) it's better to return/propagate Err instead of panicking with `.expect`